src/components/radio-button/radio-button.config.js

```diff
@@ -37,100 +38,94 @@ module.exports = {
   context: {
     prefix,
   },
-  variants: [
-    {
-      name: 'default',
-      label: 'Radio button group',
-      context: {
-        selectedValue: 'red',
-        group: 'radio-button',
-        items: items(),
-      },
-    },
-    {
-      name: 'horizontal-bottom',
-      label: 'Radio button group (Label at bottom)',
-      context: {
-        direction: 'bottom',
-        selectedValue: 'red',
-        group: 'radio-button',
-        items: items(),
-      },
-    },
-    {
-      name: 'horizontal-left',
-      label: 'Radio button group (Label at left)',
-      context: {
-        direction: 'left',
-        selectedValue: 'red',
-        group: 'radio-button',
-        items: items(),
-      },
-    },
-    {
-      name: 'horizontal-top',
-      label: 'Radio button group (Label at top)',
-      context: {
-        direction: 'top',
-        selectedValue: 'red',
-        group: 'radio-button',
-        items: items(),
-      },
-    },
-    {
-      name: 'vertical',
-      label: 'Vertical radio button group',
-      meta: {
-        xVersionOnly: true,
-      },
-      context: {
-        selectedValue: 'red',
-        group: 'radio-button--vertical',
-        vertical: true,
-        items: items(),
-      },
-    },
-    {
-      name: 'vertical-bottom',
-      label: 'Vertical radio button group (Label at bottom)',
-      meta: {
-        xVersionOnly: true,
-      },
-      context: {
-        direction: 'bottom',
-        selectedValue: 'red',
-        group: 'radio-button--vertical',
-        vertical: true,
-        items: items(),
-      },
-    },
-    {
-      name: 'vertical-left',
-      label: 'Vertical radio button group (Label at left)',
-      meta: {
-        xVersionOnly: true,
-      },
-      context: {
-        direction: 'left',
-        selectedValue: 'red',
-        group: 'radio-button--vertical',
-        vertical: true,
-        items: items(),
-      },
-    },
-    {
-      name: 'vertical-top',
-      label: 'Vertical radio button group (Label at top)',
-      meta: {
-        xVersionOnly: true,
-      },
-      context: {
-        direction: 'top',
-        selectedValue: 'red',
-        group: 'radio-button--vertical',
-        vertical: true,
-        items: items(),
-      },
-    },
-  ],
+  variants: !componentsX
```

Comment:
Switching `variants` data statically here doesn't work with our theme switcher. We use `meta.xVersionOnly` and `meta.xVersionNotSupported` for `v10`-only or `v9`-only components/variants.

Reply:
Ah thank you. I was struggling to figure out what the opposite of the xVersionOnly was and didn't know we had the xVersionNotSupported. I will change that.

Reply:
@aledavila My apologies for the struggle - https://github.com/IBM/carbon-components/blob/v9.68.0/.github/CONTRIBUTING.md#defining-markup-with-conditionals-or-data-interpolations covers those properties, but please don't hesitate to ping me if you stumble at things like this!